CMakeLists.txt

```diff
@@ -178,7 +178,7 @@ if( BUILD_WITH_TENSILE )
   else()
     # Use the virtual-env setup and download package from specified repot:
     set( tensile_fork "ROCmSoftwarePlatform" CACHE STRING "Tensile fork to use" )
-    set( tensile_tag 1c58828cafd5f1285bf17b75ad5c04ca6c8fa88c CACHE STRING "Tensile tag to download" )
+    set( tensile_tag 61a5741c591322762f64e7bd3e6a898eb2715093 CACHE STRING "Tensile tag to download" )
```

Comment:
Note that on develop, we now get the Tensile tag from a separate file:

https://github.com/ROCmSoftwarePlatform/rocBLAS/blob/develop/tensile_tag.txt

Reply:
Got it, thanks!